lib/mocha.js

```diff
@@ -143,7 +144,9 @@ Mocha.prototype.reporter = function(reporter, reporterOptions){
   } else {
     reporter = reporter || 'spec';
     var _reporter;
-    try { _reporter = require('./reporters/' + reporter); } catch (err) {}
+    // Try to load a built-in reporter.
+    if (reporters[reporter]) _reporter = reporters[reporter];
```

Comment:
Worth pointing out here is that Browserify [doesn't do dynamic requires](https://github.com/substack/node-browserify/issues/377), so we have to switch to a static approach here. I assumed that `reporter` is always in lowercase, not sure if that's a safe assumption or if we want to implement a more robust approach to grabbing built-in reporters here (spitballing here: e.g. always normalize to `kebab-case`)?